src/audio/volume/volume_hifi3.c

```diff
@@ -63,13 +63,13 @@ static void vol_s24_to_s24_s32(struct comp_dev *dev, struct audio_stream *sink,
 {
 	struct vol_data *cd = comp_get_drvdata(dev);
 	ae_f32x2 in_sample = AE_ZERO32();
-	ae_f32x2 out_sample;
-	ae_f32x2 volume;
+	ae_f32x2 out_sample = AE_ZERO32();
```

Comment:
doesn't seem to be needed

Reply:
this is to avoid KW scan errors, such as
![image](https://user-images.githubusercontent.com/93516384/162350227-f088e027-eed8-4623-bd6c-c643463ea00f.png)

Reply:
are you sure KW is complaining about any of the variables that I pointed out?

Reply:
yes, the inu/outu/volume0, etc are all critical issues.

Reply:
sorry, could you please point out specifically which of my comments contradicts KW? This specific comment is for `out_sample` which I expect KW to be able to realise, that its initialisation is redundant. But your screenshot is for `inu` for `vol_s16_to_s16()` at line 259 and if you compare with the current version, it has changed. Line 280 from your KW screenshot above isn't there any more, so your KW results are outdated. Could you re-run KW with the current code base? Thanks

Reply:
The difficulty here is the HIFI intrinsic (which looks like a load operation). I'm happy to merge here as we are not all experts on the intrinsics and the compiler will optimise out the extra assignment anyway.

Reply:
> 

sorry, I don't have the access because nobody help to approve my requirement, so I can't provide the KW scan results. This PR is related to https://github.com/thesofproject/sof/pull/5653, that Adrian told me there are some critical issues caused by those uninitialized variables, and sent those screenshots to me. That PR is for stable-v2.1 branch, and the same modification should merge to main branch.